### PR TITLE
Implemented list and find commands for employees and tasks

### PR DESCRIPTION
**Pull request by Denniszedead**

### CODE REVIEW

File: src/main/java/manageezpz/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -69,11 +71,17 @@ public Command parseCommand(String userInput) throws ParseException {
         case ClearCommand.COMMAND_WORD:
             return new ClearCommand();
 
-        case FindCommand.COMMAND_WORD:
-            return new FindCommandParser().parse(arguments);
+        case FindTaskCommand.COMMAND_WORD:
+            return new FindTaskCommandParser().parse(arguments);
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/logic/parser/AddressBookParser.java:75]: Good use of abstraction and feature implementation!

File: src/main/java/manageezpz/logic/parser/FindEmployeeCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        if (argMultimap.isPrefixExist(PREFIX_NAME)) {
+            String nameArgumentString = argMultimap.getValue(PREFIX_NAME).get().trim();
+            String[] nameArguments = nameArgumentString.split("\\s+");
+            names = checkIfAllNamesValid(nameArguments);
+        }
+        return names;
+    }
+
+    private List<String> checkIfAllNamesValid(String[] nameArguments) {
+        List<String> names = null;
```
Comment (aaron-ljx) [inline src/main/java/manageezpz/logic/parser/FindEmployeeCommandParser.java:59]: not a good practice to use null

File: src/test/java/manageezpz/logic/commands/FindEmployeeCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+import org.junit.jupiter.api.Test;
+
+import manageezpz.commons.core.Messages;
+import manageezpz.model.Model;
+import manageezpz.model.ModelManager;
+import manageezpz.model.UserPrefs;
+import manageezpz.model.person.Person;
+import manageezpz.model.person.PersonMultiplePredicate;
+
+class FindEmployeeCommandTest {
```
Comment (ChanWeiJie) [inline src/test/java/manageezpz/logic/commands/FindEmployeeCommandTest.java:23]: Thanks for including the test cases for the functions!

File: src/main/java/manageezpz/logic/parser/FindEmployeeCommandParser.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        if (!isValid) {
+            addErrorMessage(FindEmployeeCommand.INVALID_NAME);
+        } else {
+            names = Arrays.asList(nameArguments);
+        }
+        return names;
+    }
+
+    private String getPersonPhone(ArgumentMultimap argMultimap) {
+        String phone = null;
```
Comment (dannytayjy) [inline src/main/java/manageezpz/logic/parser/FindEmployeeCommandParser.java:70]: Perhaps better to create String phone as an empty String
i.e. String phone = ""

**Review by ChanWeiJie — APPROVED** [review]
LGTM! Thanks Dennis.

**Pull request merged**